SDKGenerator.njsproj

```diff
@@ -101,6 +101,11 @@
     <OutputPath>bin\</OutputPath>
     <ScriptArguments>csharp-unity-gameserver=..\sdks\PlayFabGameServer -apiSpecPath -buildIdentifier UnitySDK_manual</ScriptArguments>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Python' ">
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\</OutputPath>
+    <ScriptArguments>python=..\sdks\Python -apiSpecPath -buildIdentifier Python_manual</ScriptArguments>
```

Comment:
Rename the target directory to "PythonSdk" to match the primary repo name.
(other non-matching names are older, and should be updated someday)
This line should be:
pythonsdk=..\sdks\PythonSdk -apiSpecPath -buildIdentifier PythonSdk_Manual